harvard-cape-peninsula-university-of-technology.csl

```diff
@@ -109,8 +109,10 @@
   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
     <layout prefix="(" suffix=")" delimiter="; ">
       <group>
-        <text macro="author-short"/>
-        <text macro="year-date" prefix=", "/>
+        <group delimiter=", "/>
```

Comment:
you can't have the / at the end here. should be  ```<group delimiter=", ">```
It would also be great if you could delete ```<group>``` in line 111 and ```</group>``` in line 116: these don't do anything now. Same update instructions as by @rmzelle above.